server/api-service/lowcoder-server/src/main/java/org/lowcoder/api/framework/filter/ThrottlingFilter.java

```diff
@@ -1,37 +1,41 @@
 package org.lowcoder.api.framework.filter;
 
-import com.google.common.util.concurrent.RateLimiter;
-import jakarta.annotation.PostConstruct;
-import lombok.Getter;
-import lombok.extern.slf4j.Slf4j;
+import static java.util.Collections.emptyMap;
+import static org.lowcoder.api.framework.filter.FilterOrder.THROTTLING;
+import static org.lowcoder.sdk.exception.BizError.REQUEST_THROTTLED;
+import static org.lowcoder.sdk.util.ExceptionUtils.ofError;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Supplier;
+
+import javax.annotation.Nonnull;
+
 import org.lowcoder.sdk.config.dynamic.ConfigCenter;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 import org.springframework.core.Ordered;
 import org.springframework.http.server.reactive.ServerHttpRequest;
 import org.springframework.stereotype.Component;
 import org.springframework.web.server.ServerWebExchange;
 import org.springframework.web.server.WebFilter;
 import org.springframework.web.server.WebFilterChain;
-import reactor.core.publisher.Mono;
-
-import javax.annotation.Nonnull;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Supplier;
 
-import static java.util.Collections.emptyMap;
-import static org.lowcoder.api.framework.filter.FilterOrder.THROTTLING;
-import static org.lowcoder.sdk.exception.BizError.REQUEST_THROTTLED;
-import static org.lowcoder.sdk.util.ExceptionUtils.ofError;
+import com.google.common.util.concurrent.RateLimiter;
 
+import jakarta.annotation.PostConstruct;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import reactor.core.publisher.Mono;
 
+@ConditionalOnExpression("${default.api-rate-limit:0} > 0")
 @SuppressWarnings("UnstableApiUsage")
 @Slf4j
 @Component
 public class ThrottlingFilter implements WebFilter, Ordered {
 
-	@Value("${default.apiRateLimit:50}")
+	@Value("${default.api-rate-limit}")
```

Comment:
@ludomikula : One question just for clarity/understanding. 
Was the value (`apiRateLimit`) never getting picked from the spring yml config file before? Because I see that the spring config has `api-rate-limit` instead of `apiRateLimit`. 
I think it was always defaulting to 50 and not reading from yml before.
The current change has corrected the config name which is ok, ofc.
Please correct my understanding if wrong.

Reply:
Unfortunately yes - it works if you map a bean with variables denoted in camel case, but not when you use it in `@Value` or a SPeL expression.